docs/learn/learn-runtime-upgrades.md

```diff
@@ -120,38 +120,14 @@ Runtime upgrades don't require any actions by a nominator, though it is always e
 up-to-date and participate with the latest runtime upgrade motions and releases while keeping an eye
 on how the nodes on the network are reacting to a new upgrade.
 
-## Monitoring Runtime Changes
+## OpenGov Runtime Upgrade Monitoring
```

Comment:
The section title can stay the same (for backward compatibility of shared links). 

```suggestion
## Monitoring Runtime Changes
```